src/pyb2b/xml/ForecastFlightPlanRequest.xml

```diff
@@ -0,0 +1,31 @@
+ <flig:FlightListByAerodromeRequest xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
+ xmlns:flig="eurocontrol/cfmu/b2b/FlightServices">
+    <!--Optional:-->
+    <endUserId>bcramet</endUserId>
+    <sendTime>{send_time:%Y-%m-%d %H:%M:%S}</sendTime>
+    <dataset>
+    <type>FORECAST</type>
+    </dataset>
+    <includeProposalFlights>true</includeProposalFlights>
+    <includeForecastFlights>true</includeForecastFlights>
+    <trafficType>DEMAND</trafficType>
+    <!--Optional:-->
+    <trafficWindow>
+    <wef>{day} 00:00</wef>
```

Comment:
This could be smarter, if `day` is a timestamp?
```
{day:%Y-%m-%d %H:%M}
```

Reply:
yes, then I will update it to be a start and an end (as you have done for the flight list)